src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/Event.scala

```diff
@@ -0,0 +1,257 @@
+                 br_type:                  Option[String],
+                 br_renderengine:          Option[String],
+                 br_lang:                  Option[String],
+                 br_features_pdf:          Option[Boolean],
+                 br_features_flash:        Option[Boolean],
+                 br_features_java:         Option[Boolean],
+                 br_features_director:     Option[Boolean],
+                 br_features_quicktime:    Option[Boolean],
+                 br_features_realplayer:   Option[Boolean],
+                 br_features_windowsmedia: Option[Boolean],
+                 br_features_gears:        Option[Boolean],
+                 br_features_silverlight:  Option[Boolean],
+                 br_cookies:               Option[Boolean],
+                 br_colordepth:            Option[String],
+                 br_viewwidth:             Option[Int],
+                 br_viewheight:            Option[Int],
+                 os_name:                  Option[String],
+                 os_family:                Option[String],
+                 os_manufacturer:          Option[String],
+                 os_timezone:              Option[String],
+                 dvce_type:                Option[String],
+                 dvce_ismobile:            Option[Boolean],
+                 dvce_screenwidth:         Option[Int],
+                 dvce_screenheight:        Option[Int],
+                 doc_charset:              Option[String],
+                 doc_width:                Option[Int],
+                 doc_height:               Option[Int],
+                 tr_currency:              Option[String],
+                 tr_total_base:            Option[Double],
+                 tr_tax_base:              Option[Double],
+                 tr_shipping_base:         Option[Double],
+                 ti_currency:              Option[String],
+                 ti_price_base:            Option[Double],
+                 base_currency:            Option[String],
+                 geo_timezone:             Option[String],
+                 mkt_clickid:              Option[String],
+                 mkt_network:              Option[String],
+                 etl_tags:                 Option[String],
+                 dvce_sent_tstamp:         Option[Instant],
+                 refr_domain_userid:       Option[String],
+                 refr_device_tstamp:       Option[Instant],
+                 derived_contexts:         Contexts,
+                 domain_sessionid:         Option[String],
+                 derived_tstamp:           Option[Instant],
+                 event_vendor:             Option[String],
+                 event_name:               Option[String],
+                 event_format:             Option[String],
+                 event_version:            Option[String],
+                 event_fingerprint:        Option[String],
+                 true_tstamp:              Option[Instant]) {
```

Comment:
I think we can and I agree about timing.

The only one object that I had from the beginning is that Analytics SDK is supposed to support more than one version of enriched event (and it never did).

Also, if we're going to change modules structure - it would make more sense to remove manifest functionality along with AWS SDK altogether.

Reply:
What would be your proposal for module/dependency tree?

Something like following?

1. Plain `Event` - one module
2. TSV Decoder and JSON Encoder - another module

Reply:
I think so yes, we could even create a module with a tsv encoder at some point (or add it to the second)